scripts/install_java.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 pushd .
 
+printf "Acquire::http::Proxy \"http://$PROXY_UNAME:$PROXY_PASSWORD@L02PIPROXY01.corp.local:80\";" > /etc/apt/apt.conf
```

Comment:
I'd prefer to put this in the `apt.conf.d` folder in `/etc/apt/apt.conf.d/00AscenaProxy`

... we may also need to handle HTTPS proxy settings.

Also, this script will run as part of the provisioner, which means that it's going to run regardless of what builder we use.  This means that we don't necessarily want this proxy to be configured everywhere.  We should make the proxy itself and environment variable as well, and then check for it using Bash before we write it out.